digitalocean/Manager.py

```diff
@@ -127,11 +127,13 @@ def get_all_images(self):
         images = self.get_images()
         return images
 
-    def get_image(self, image_id):
+    def get_image(self, image_id_or_slug, use_slug=False):
         """
-            Return a Image by its ID.
+            Return a Image by its ID/Slug.
         """
-        return Image.get_object(api_token=self.token, image_id=image_id)
+        return Image.get_object(
+            self.token, image_id_or_slug, use_slug=use_slug
```

Comment:
is this backwards-compatible?
Please define `api_token=` and `image_id_or_slug=` as it used to be to keep things simple.